forge-gui/res/cardsfolder/upcoming/it_doesnt_add_up.txt

```diff
@@ -0,0 +1,8 @@
+Name:It Doesn't Add Up
+ManaCost:3 B B
+Types:Instant
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | TgtPrompt$ Choose target creature card in your graveyard | ValidTgts$ Creature.YouOwn | SubAbility$ DBSuspect | SpellDescription$ Return target creature card from your graveyard to the battlefield.
+SVar:DBSuspect:DB$ AlterAttribute | Attributes$ Suspected | Defined$ Targeted | StackDescription$ SpellDescription | SpellDescription$ Suspect it. (It has menace and can't block.)
```

Comment:
I don't think altering the old card will work

Reply:
It did work in my testing but I guess using a remember would be better ?

Reply:
hmn no big deal then